setup.py

```diff
@@ -0,0 +1,34 @@
+import distutils.core
+from distutils.command.install import install
+import subprocess
+
+
+class PyenvInstall(install):
+    def run(self):
+        print(subprocess.check_output(['bash', 'bin/pyenv-installer']))
+
+with open('README.rst') as file:
+    long_description = file.read()
+
+distutils.core.setup(
+    version='20150113',
```

Comment:
If I once release a new version of pyenv, should I update this line and then release to PyPI too?